app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -222,10 +241,11 @@ class @AgentEditPage
     @updateFromEditors()
     Utils.handleDryRunButton(e.target)
 
-  formatAgentForSelect = (agent) ->
-    originalOption = agent.element
-    description = agent.element[0].title
-    '<strong>' + agent.text + '</strong><br/>' + description
+  formatAgentForSelect = (data) ->
+    if data.title == ''
```

Comment:
I believe so.  It's the value of `agent.html_description.lines.first.strip`, like `<p>The Website Agent scrapes a website, XML document, or JSON feed and creates Events based on the results.</p>\n`.